src/adapters/WHEPAdapter.ts

```diff
@@ -178,8 +181,13 @@ export class WHEPAdapter implements Adapter {
           sdp: answer
         });
       } else if (response.status === 400) {
-        this.log(`server does not support client-offer, need to reconnect`);
-        this.whepType = WHEPType.Server;
```

Comment:
We still need to support WHEP endpoints that does not offer client-side offers, at least until that part has been settled in the WHEP standard